### PR TITLE
Patient info modal

### PR DESCRIPTION
**Pull request by liangfe5**
image

Modal popup.

### CODE REVIEW

File: get-docker.sh [hunk 1]
```diff
@@ -0,0 +1,630 @@
+#!/bin/sh
```
Comment (nikita-sh) [inline get-docker.sh:1]: Do we need this file commited?

File: src/client/src/components/patientmodal/Modal.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import './Modal.css';
+
+export const Modal = ({ patient_info, handleClose, show,}) => {
+  const showHideClassName = show ? "modal display-block" : "modal display-none";
+  console.log(patient_info);
```
Comment (nikita-sh) [inline src/client/src/components/patientmodal/Modal.js:5]: Remove `console.log`

**Review by nikita-sh — CHANGES_REQUESTED** [review]
Could we also reduce the size of the patient modals/cards just a bit? Not necessary for this milestone but it feels like they're a tad big. Otherwise, nice job! Really digging how it's coming along.

File: package.json [hunk 3]
```diff
@@ -16,9 +16,22 @@
...
+    "react-bootstrap": "^2.0.2",
+    "react-dom": "^17.0.2",
+    "react-modal": "^3.14.3",
+    "react-modal-bootstrap": "^1.1.1",
+    "react-native": "^0.66.2",
+    "react-native-cli": "^2.0.1",
+    "react-native-modal": "^13.0.0",
+    "react-native-modals": "^0.22.3",
+    "react-native-web": "^0.17.5",
+    "react-router-native": "^6.0.0"
```
Comment (vicchig) [inline package.json:35]: @liangfe5 

If we are not using these dependencies, could you please remove them? If we need them, please move them into package.json and package-lock.json in the client folder.

**Review by nikita-sh — CHANGES_REQUESTED** [review]
Once the react dependencies that need to stay in are moved to the client folder like @vicchig mentioned, LGTM!

**Review by vicchig — APPROVED** [review]

**Review by nikita-sh — APPROVED** [review]

**Pull request merged**